data.sql

```diff
@@ -16,3 +16,47 @@ VALUES ('Pikachu', '2021-01-07', 15.04, false, 1);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Devimon', '2017-05-12', 11.0, true, 5);
 
+---- TASK 2-------
+
+-- Inserting the first animal (Charmander) with a negative weight
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Charmander', '2020-02-08', -11.0, false, 0, 'Fire-type');
+
+-- Inserting the second animal (Plantmon) with a negative weight
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Plantmon', '2021-11-15', -5.7, true, 2, 'Grass-type');
+
+-- Inserting the third animal (Squirtle) with a negative weight
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Squirtle', '1993-04-02', -12.13, false, 3, 'Water-type');
+
+-- Inserting the fourth animal (Angemon) with a negative weight
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Angemon', '2005-06-12', -45.0, true, 1, 'Holy-type');
+
+-- Inserting the fifth animal (Boarmon)
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Boarmon', '2005-06-07', 20.4, true, 7, 'Ground-type');
+
+-- Inserting the sixth animal (Blossom)
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Blossom', '1998-10-13', 17.0, true, 3, 'Grass-type');
+
+-- Inserting the seventh animal (Ditto)
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Ditto', '2022-05-14', 22.0, true, 4, 'Normal-type');
```

Comment:
- [ ] Please always double-check your work before submitting your project for review. I tried to see what you did and I got an error related to `id`, because this ID column is not null and in order to resolve the problem, a user must add the unique ID himself or herself. 

![image](https://github.com/stino-x/vet-clinic-psql/assets/66476329/e1a2191c-c612-4bef-a5fa-367ecaa056f8)